### PR TITLE
Form elements react components - Select

### PR DESCRIPTION
**Pull request by matikrk**
* select component, tests, examples
* textarea - delete type property
* fix warnings in pages

### CODE REVIEW

File: src/components/form-elements/Select.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+const VALIDATION = {
+  VALID: true,
+  INVALID: false,
+  UNDEFINED: undefined
+};
+
+
+const Select = props => {
+  const {
+    valid = VALIDATION.UNDEFINED,
```
Comment (aju) [inline src/components/form-elements/Select.jsx]: maybe state ?

File: src/components/form-elements/Select.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    fullWidth,
+    value,
+    options = [],
+    ...additionalProps
+  } = props;
+
+  const selectClass = classnames('sg-select', {
+    'sg-select--valid': valid === VALIDATION.VALID,
```
Comment (aju) [inline src/components/form-elements/Select.jsx]: It should not be possible to have valid and invalid on same element

File: src/components/form-elements/pages/text-input.jsx [hunk 3]
```diff
@@ -8,10 +8,10 @@ const textInputs = () => <div>
     <TextInput placeholder="placeholder"/>
   </DocsBlock>
   <DocsBlock info="Valid">
-    <TextInput placeholder="placeholder" valid={true} value="This is valid example"/>
+    <TextInput placeholder="placeholder" valid={true} value="This is valid example" onChange={() => undefined}/>
```
Comment (aju) [inline src/components/form-elements/pages/text-input.jsx]: why onChange empty?

File: src/components/form-elements/Select.jsx (hunk 2 above)
Comment (matikrk) [inline src/components/form-elements/Select.jsx]: So it isn't :)

File: src/components/form-elements/pages/text-input.jsx (hunk 3 above)
Comment (matikrk) [inline src/components/form-elements/pages/text-input.jsx]: to get off warnings when creating our examples

Comment (aju) [inline src/components/form-elements/pages/text-input.jsx]: then make this function optional

File: src/components/form-elements/Select.jsx (hunk 1 above)
Comment (aju) [inline src/components/form-elements/Select.jsx]: lets leave it as it is

**Pull request merged**